java/src/main/java/org/rocksdb/RocksDB.java

```diff
@@ -512,6 +592,36 @@ public void put(final ColumnFamilyHandle columnFamilyHandle,
         0, value.length, columnFamilyHandle.nativeHandle_);
   }
 
+  /**
+   * Set the database entry for "key" to "value" for the specified
+   * column family.
+   *
+   * @param columnFamilyHandle {@link org.rocksdb.ColumnFamilyHandle}
+   *     instance
+   * @param writeOpts {@link org.rocksdb.WriteOptions} instance.
+   * @param key The specified key to be inserted
```

Comment:
Can you please specify that RocksDB won't keep any reference to the data but it will perform a copy?
This is needed in order to re-use the byte[].

The same comment applies to read-only methods